extern/recastnavigation/Detour/Include/DetourAssert.h

```diff
@@ -22,7 +22,15 @@
 // Note: This header file's only purpose is to include define assert.
 // Feel free to change the file and include your own implementation instead.
 
-#ifdef NDEBUG
+#ifdef __clang_analyzer__
+
+#	include <assert.h>
+#	define dtAssert(expression) \
+		{ \
+			assert(expression); \
+		}
+
+#elif defined NDEBUG
```

Comment:
Will we try to get this upstreamed if it works?

Reply:
Unfortunately, it does not work - there is still 23 warnings.

Reply:
ok ;)